tools/tgs4_scripts/PreCompile.bat

```diff
@@ -0,0 +1,14 @@
+@echo off
+set TG_BOOTSTRAP_CACHE=%cd%
+IF NOT "%1" == "" (
```

Comment:
Gah! The working version of TGS3 doesn't set the CWD to the event folder either.
```suggestion
cd /D "%~dp0"
IF NOT "%1" == "" (
```